sources.md

```diff
@@ -0,0 +1,11 @@
+# sources
+
+## SNIRH database
+
+[SNIRH data](https://snirh.pt/snirh/_dadosbase/site/simplex.php?OBJINFO=DADOS&FILTRA_BACIA=138&FILTRA_COVER=920123704&FILTRA_SITE=920685260)
+
+We have data for monthly mean air temperature and monthly precipitation from 01/10/1980 to 01/02/2022.
+With these data we can understand the correlation between precipitation and air temperature and answer the main question of client has.
+
+It seems good and we could choose the monthly average temperature and monthly precipitation value
+
```

Comment:
[wordclim](https://www.worldclim.org/data/monthlywth.html)

This is another source of data on precipitation and air temperature that we can use